wireless_modules/mqtt_client.py

```diff
@@ -22,8 +24,24 @@ def connect_and_subscribe(self, topics_to_subscribe, callback_func):
         self.client.set_callback(callback_func)
 
         # Connect to MQTT broker
-        self.client.connect()
+        connection_success = False
+        while not connection_success:
+            connection_success = True
```

Comment:
Bit pedantic, but since we already have the class variable `self.connected`, we can probably just use that here.